omod-2.2/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_2/PersonResource2_2.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+package org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs2_2;
+
+import org.openmrs.module.webservices.rest.web.representation.Representation;
+import org.openmrs.module.webservices.rest.web.resource.impl.DelegatingResourceDescription;
+import org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs1_8.PersonResource1_8;
+
+public class PersonResource2_2 extends PersonResource1_8 {
+	
+	@Override
+	public DelegatingResourceDescription getRepresentationDescription(Representation rep) {
+		DelegatingResourceDescription description = super.getRepresentationDescription(rep);
+		
+		if(description != null) {
+			description.addProperty("causeOfDeathNonCoded");
```

Comment:
I think you should check the representation type so that you don't add the field for the RefRepresentation

Reply:
But it is the same way done in 1_11. There is is also just checked if it is != null.

Shall I yet do the instance of check?

Reply:
It was just an example to follow, you shouldn't copy everything literally, context and the property you're adding determines, in this case causeOfDeath is null in most cases for most patients, therefore it's not key information, so you should not include it for the RefRepresentation which  is a very minimal form.